### PR TITLE
json type column for mysql 5.7

### PR DESCRIPTION
**Pull request by cedriclombardot**

### CODE REVIEW

File: src/Propel/Generator/Builder/Om/ObjectBuilder.php [hunk 1]
```diff
@@ -968,6 +968,21 @@ protected function addObjectAccessorBody(&$script, Column $column)
...
+        $this->addDefaultAccessorOpen($script, $column);
+        $this->addJsonAccessorBody($script, $column);
+        $this->addDefaultAccessorClose($script);
+    }
+
+    protected function addJsonAccessorBody(&$script, Column $column)
+    {
+        $clo = $column->getLowercasedName();
+        $script .= "
+        return json_decode(\$this->$clo);";
```
Comment (marcj) [inline src/Propel/Generator/Builder/Om/ObjectBuilder.php]: What if the column is empty or null?

Comment (cedriclombardot) [inline src/Propel/Generator/Builder/Om/ObjectBuilder.php]: Same :

````
php > var_dump(json_decode(json_encode("")));
string(0) ""
php > var_dump(json_decode(json_encode(NULL)));
NULL
`````

Comment (marcj) [inline src/Propel/Generator/Builder/Om/ObjectBuilder.php]: Well, $this->$clo is null and you call thus `json_decode(null);`. I don't see a json_encode in that line.

Comment (cedriclombardot) [inline src/Propel/Generator/Builder/Om/ObjectBuilder.php]: For me it also working 

`````
php > echo json_decode(null);
php > var_dump(json_decode(null));
php shell code:1:
NULL
`````

But if u prefer i can add one if

Comment (marcj) [inline src/Propel/Generator/Builder/Om/ObjectBuilder.php]: Ok, great, then this is not an issue :)

**Comment by marcj** [comment]
Can you also add a little test that covers this feature?

**Comment by marcj** [comment]
Great stuff, thanks :)

**Pull request merged**